mixins/form.jsx

```diff
@@ -342,13 +347,23 @@ module.exports = {
     if (!valid) {
       return;
     }
+    var description = this.getIntlMessage("mozilla_donation");
+    var handlerDesc = "";
     submitProps = this.buildProps(props);
+    if (submitProps.frequency === "monthly") {
+      description = this.getIntlMessage("mozilla_monthly_donation");
+      handlerDesc = this.getIntlMessage("monthly");
+    }
+
     var locale = this.props.locales[0];
     var currency = this.state.currency && this.state.currency.code;
     var handler = StripeCheckout.configure({
       // Need to get this from .env
       key: process.env.STRIPE_PUBLIC_KEY,
       image: '',
+      zipCode: true,
+      billingAddress: true,
+      locale: locale,
```

Comment:
Moving locale into the config and not the open. This is fairly semantic, but I'm going for configure setsup the layout and context of the widget, and open fills in the layout and context with data.